tests/msc2716_test.go

```diff
@@ -876,6 +876,77 @@ func TestImportHistoricalMessages(t *testing.T) {
 					},
 				})
 			})
+
+			t.Run("Backfill still works after many batches are imported", func(t *testing.T) {
```

Comment:
Make it obvious that these batches are connected together

```suggestion
			t.Run("Backfill still works after many contiguous batches are imported", func(t *testing.T) {
```